tests/test_mapped_estimators.py

```diff
@@ -78,7 +80,7 @@ def test_one_sided_estimates():
         assert np.std(mapped_w_F) < eps
 
         # ... and estimated_delta_f should be == exact_delta_f
-        estimated_delta_f = EXP(mapped_w_F)[0]
+        estimated_delta_f = exp(mapped_w_F)[DG_KEY]
```

Comment:
nit: might prefer qualified `pymbar.exp` here to avoid confusion with the exponential function

Reply:
updated in https://github.com/proteneer/timemachine/pull/1492/commits/cb46294fd0c28d33d45cb8b4eed06883edcee0db